website/cue/reference/components/sources/pipe.cue

```diff
@@ -0,0 +1,121 @@
+package metadata
+
+components: sources: pipe: {
+	title: "PIPE"
```

Comment:
"Pipe" is not an acronym, so should only be capitalized instead of all-capitals.